File: lektionen/javascript/Kata/k6_split_strings.js
function solution(str){
  let re = /(\w{2})/ig
  let arr = str.split(re).filter(x => x != "");
  if (str.length % 2 == 0) {
    return arr;
  }
  else {
    arr[arr.length - 1] =  arr[arr.length - 1] + "_";
    return arr;
  }  
}

function solution(str){
  let re = /(\w{2})/ig
  let arr = str.split(re).filter(x => x != "");
  if (str.length % 2 == 0) {
    return arr;
  }
  else {
    arr.push(arr.pop() + "_");
    return arr;
  }  
}

File: lektionen/python/Kata/k6_split_strings.py
def solution(s):
  arr = []
  if len(s) % 2 == 0:
    for i in range(0, len(s), 2):
      arr.append(s[i:i+2])
  else:
    s+="_"
    for i in range(0, len(s), 2):
      arr.append(s[i:i+2])

  return arr

print(solution('abcdefg'))